kalamine/data/scan_codes.yaml

```diff
@@ -1,66 +1,66 @@
 klc:
-  spce: '39	SPACE'
+  spce: '39'
 
   # digits
-  ae01: '02	1'
-  ae02: '03	2'
-  ae03: '04	3'
-  ae04: '05	4'
-  ae05: '06	5'
-  ae06: '07	6'
-  ae07: '08	7'
-  ae08: '09	8'
-  ae09: '0a	9'
-  ae10: '0b	0'
+  ae01: '02'
+  ae02: '03'
+  ae03: '04'
+  ae04: '05'
+  ae05: '06'
+  ae06: '07'
+  ae07: '08'
+  ae08: '09'
+  ae09: '0a'
+  ae10: '0b'
```

Comment:
I think TOML allows hex integers, which would be cool to see to avoid confusion. But I’ll trust your judgement and won’t block on this.

Reply:
We use these in an `str` only so it avoids type conversion